### ISSUE TITLE
Don’t Talk To The Police

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/nelsonic.github.io#833]
This is one of the most insightful (into the legal/penal system) lectures I’ve watched: 
https://youtu.be/d-7o9xYp7eE

**Comment by iteles** [comment]
Fascinating 😳

**Comment by nelsonic** [comment]
Yeah, it's *bonkers* that what you say can be used _against_ you but not _for_ you. 

> "_You have the **right** to **remain silent**. **Anything you say** can be **used against you** in court. You have the right to talk to a **lawyer** for advice **before** we ask you any questions. You have the right to have a lawyer with you during questioning. If you cannot afford a lawyer, one will be appointed for you before any questioning if you wish. If you decide to answer questions now without a lawyer present, you have the right to **stop answering** at any time._" 
~ https://en.wikipedia.org/wiki/Miranda_warning

_Everyone_ in 'Murica should watch this.
